x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/prebuilt_rules_preview.cy.ts

```diff
@@ -240,6 +240,7 @@ describe('Detection rules, Prebuilt Rules Installation and Update workflow', ()
     query: 'process where process.name == "regsvr32.exe"',
     index: ['winlogbeat-*', 'logs-endpoint.events.*'],
     filters,
+    alert_suppression: undefined,
   });
 
   const THREAT_MATCH_INDEX_PATTERN_RULE = createRuleAssetSavedObject({
```

Comment:
Nit: INDICATOR_MATCH_INDEX_PATTERN_RULE?

Reply:
This naming is part of existing code base, so I didn't touch it. If we would like to rename all mentions of threat_match to indicator_match I would suggest to do in a separate PR for all of variables at the same time, so it won't be named differently in different parts of code base.

Worth to note, rule type field `threat_match` and schema fields(threat*) can't be change